bodybank/character_generator/models.py

```diff
@@ -1,3 +1,4 @@
+from ssl import OP_ENABLE_MIDDLEBOX_COMPAT
```

Comment:
What do we need this for? Delete?

Reply:
Oh gosh! This is awkward . . . I am not sure what that is used for . . .

Reply:
how did it even get in, IDE magic?